Generator/Mustache.php

```diff
@@ -19,7 +19,7 @@ class Mustache
      *
      * @return mixed
      */
-    public static function replace($string, array $parameters)
+    public static function replace(string $string, array $parameters)
```

Comment:
The return type is `?string`. From the [doc](http://php.net/manual/en/function.preg-replace-callback.php):

> mixed preg_replace_callback ( mixed $pattern , callable $callback , mixed $subject [, int $limit = -1 [, int &$count ]] )
>
> ## Return values
> preg_replace_callback() returns an array if the subject parameter is an array, or a string otherwise. On errors the return value is NULL.

Moreover, this should be simplified to `string`. That way if an error occurs, and `preg_replace_callback` returns `null`, then a fatal exception is trigerred.